packages/synapse-react-client/src/synapse-client/SynapseClient.ts

```diff
@@ -1117,6 +1118,22 @@ export const getEntityBundleV2 = <T extends EntityBundleRequest>(
   )
 }
 
+/**
+ * Get the URL of the preview file associated with the current version of a FileEntity.
+ * https://rest-docs.synapse.org/rest/GET/entity/id/filepreview.html
+ */
+export const getEntityPreview = (
```

Comment:
Am I seeing this right that this is unused? Guessing you were going to use this but then just decided to take the easier approach and just use `ImageFileHandle` and not use the preview?